gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-ui.yaml

```diff
@@ -180,6 +180,58 @@ paths:
                   $ref: "#/components/schemas/Theme"
           default:
             $ref: "#/components/responses/Error"
+
+    /environments/{envId}/ui/themes/_current:
```

Comment:
maybe check with Flo but I don't know if we need the _ here same for default 🤷‍♂️

Reply:
I can check with him when he's back. 
As far as my reasoning, I used the `_` because without it, `"default"` or `"current"` would seem to be the literal ids or names of a theme. But 🤷‍♀️ , we can always change it later.